pmc-rs/tests/pmc.rs

```diff
@@ -35,7 +35,7 @@ fn test_set_counter() {
 fn test_counter_bad_name() {
     let err = CounterBuilder::default()
         .attach_to(vec![0])
-        .allocate("inst_retired.any")
```

Comment:
Huh, this is weird - maybe a bad copy/paste when I rewrote the public API?

Reply:
Yeah, probably.